Gopkg.lock

```diff
@@ -1,6 +1,18 @@
 # This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
```

Comment:
Why is this file not ignored? I don't think it is necessary to commit this file.

Reply:
This file ensures consistent versions of dependencies, which we want to have